### PR TITLE
feat: adding cert manager for tls support

### PR DESCRIPTION
**Pull request by AngeloB-AIS**
## Description

Adding TLS support with cert-manager

## Related Issue

[INFRA-21](https://linear.app/ai-squared/issue/INFRA-21/deploy-multiwoven-in-aks-via-helm-chart)

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] New Connector (Destination or Source Connector)
- [ ] Breaking change (fix or feature that would impact existing functionality)
- [ ] Styling change
- [ ] Documentation update
- [x] Refactoring
- [ ] Chore

## How Has This Been Tested?
1. `helm repo add jetstack https://charts.jetstack.io/ --force-update`
2. `helm repo update`
3. `helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --version v1.14.5 --set installCRDs=true`
4. `helm upgrade -i multiwoven`
5. Browse to site and confirm it's secured

### CODE REVIEW

File: charts/multiwoven/values.yaml [hunk 1]
```diff
@@ -33,6 +33,8 @@ multiwovenConfig:
   temporalUiVersion: 2.23.2
   temporalVersion: 1.22.4
   temporalWorkflowThreadPoolSize: "10"
+  tlsAdminEmail: admin@squared.ai
```
Comment (RafaelOAiSquared) [inline charts/multiwoven/values.yaml:36]: The default should be something like: `admin@example.com`

File: charts/multiwoven/templates/multiwoven-config.yaml [hunk 2]
```diff
@@ -42,6 +42,7 @@ data:
   UI_HOST: {{ .Values.multiwovenConfig.uiHost | quote }}
   VITE_API_HOST: {{ .Values.multiwovenConfig.viteApiHost | quote }}
   VITE_BRAND_NAME: {{ .Values.multiwovenConfig.viteBrandName | quote }}
+  VITE_BRAND_LOGO: {{ .Values.multiwovenConfig.viteBrandLogo | quote }}
```
Comment (RafaelOAiSquared) [inline charts/multiwoven/templates/multiwoven-config.yaml:45]: Delete `VITE_BRAND_LOGO` here and delete `viteBrandLogo` from `values.yaml`

**Review by RafaelOAiSquared — CHANGES_REQUESTED** [review]
Delete the `.DS_Store` files.

File: charts/multiwoven/values.yaml (hunk 1 above)
Comment (AngeloB-AIS) [reply charts/multiwoven/values.yaml:36]: how about admin@multiwoven.com?

**Review by RafaelOAiSquared — APPROVED** [review]

**Review by subintp — APPROVED** [review]

**Pull request merged**